snakemake_executor_plugin_slurm/__init__.py

```
@@ -803,12 +812,20 @@ def cancel_jobs(self, active_jobs: List[SubmittedJobInfo]):
                 # about 30 sec, but can be longer in extreme cases.
                 # Under 'normal' circumstances, 'scancel' is executed in
                 # virtually no time.
-                scancel_command = f"scancel {jobids} --clusters=all"
+                scancel_command = f"scancel {jobids}"
+
+                # adding the --clusters=all flag, if we submitted to more than one cluster
+                # issue #397 mentions that this flag is not available in older SLURM versions,
+                # but we assume that multicluster setups will usually run on a recent version
+                # of SLURM.
+                if self._submitted_job_clusters:
+                    scancel_command += f" --clusters=all"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)